express_relay/sdk/js/src/index.ts

```diff
@@ -59,6 +59,19 @@ export function checkTokenQty(token: {
   };
 }
 
+class ChainConfig {
+  wethAddress: Address;
+  opportunityAdapterAddress: Address;
```

Comment:
does the `chainId` not also fit in here?

Reply:
Yes, but we are not using it in the sdk.